content/tracing/go.md

```diff
@@ -8,15 +8,43 @@ customnav: tracingnav
 
 ### Installation
 
-To begin tracing applications written in Go, first [install and configure the Datadog Agent](/tracing#installing-the-agent).
+To begin tracing applications written in Go you need to:
 
-Next, install the Go Tracer from the Github repository:
+ - [Install and configure the Datadog Agent](/tracing#installing-the-agent).
+
+ -  Install the Go Tracer from the Github repository:
 
 ~~~
 go get "github.com/DataDog/dd-trace-go/tracer"
 ~~~
 
-Finally, import the tracer and instrument your code!
+This Package contains Datadog's tracing client. It is used to trace requests as they flow across web servers, databases and microservices so that developers have visibility into bottlenecks and troublesome requests.
```

Comment:
> This Package contains Datadog's tracing client.

This is redundant; we're already saying "Get the Go tracer library".

> It is used to trace requests as they flow across web servers, databases and microservices so that developers have visibility into bottlenecks and troublesome requests.

It seems like you're trying to avoid saying "you", but for our Docs site, "you" is not forbidden. I've been using it a lot—it's more personal, and often results in clearer descriptions.

I think this is friendlier:

"The tracer library lets you trace requests as they flow across web servers, databases, and other services so you can identify bottlenecks and troublesome requests."